src/dfmpreview/.gitignore

```diff
@@ -0,0 +1,34 @@
+###############
+#    folder   #
+###############
+node_modules/
+out/
+
+
+###############
+#  temp file  #
+###############
+*.nupkg
+*.log
+*.tmp
+*.vsprops
+*.suo
+*.user
+*.userprefs
+*.sdf
+*.sln.ide
+*.cache
+*.sln.metaproj
+.gitattributes
+.settings
+log.txt
+debugSettings.json
+launchSettings.json
+project.lock.json
+*.sublime-workspace
+*.nuget.targets
+
+###############
+#  template   #
+###############
+src/docfx/Template/*.zip
```

Comment:
I think so, we need this if this project is a standalone project

---
In reply to: [73489606](https://github.com/dotnet/docfx/pull/558#discussion_r73489606) [](ancestors = 73489606)